server/seeder.js

```diff
@@ -63,4 +65,86 @@ const fakeSchema = {
   ],
 };
 
-module.exports = fakeSchema;
+mongoose.connect(`mongodb://${process.env.DB_HOST}:${process.env.DB_PORT}/fec`, {
+  useNewUrlParser: true,
+  useUnifiedTopology: true,
+})
+  .catch((err) => {
+    console.error.bind(console, 'connection error:', err);
+  });
+
+const db = mongoose.connection;
+
+const roomSchema = {
+  id: {
+    type: Number,
+    index: true,
+  },
+  price: Number,
+  cleaning: Number,
+  service: Number,
+  tax: Number,
+  maxGuests: Number,
+  // blackouts: [Date],
+};
+const Room = mongoose.model('Room', roomSchema);
```

Comment:
I'm curious to hear your thought process behind deciding to create the database connection and define the schema in both seeder.js and database.js!

Reply:
The idea is that the seeder script is completely modular. Nothing else has to know about the seeder, and the seeder doesn't need to know about anything else. This means nothing the seeder does can cause unintended effects elsewhere in the system.

While I could refactor to export the Room model from database.js, that would cause database.js to run, with its own connection.

I originally wrote all of this inside database.js but moved it out to avoid cross-contaminating the logics.